docs/extend/extend-apiml/api-mediation-personal-access-token.md

```diff
@@ -0,0 +1,151 @@
+4. [Invalidate all the tokens for a user](#invalidate-all-the-tokens-for-a-user)
+5. [Invalidate all the tokens for a service](#invalidate-all-the-tokens-for-a-service)
+6. [Retrieve all the tokens and rules](#retrieve-all-the-tokens-and-rules)
+7. [Evict the non-relevant tokens and rules](#evict-the-non-relevant-tokens-and-rules)
+
+The **User** is able to :
+* Generate the token
+* Validate the token
+* Invalidate a specific token
+* Invalidate all tokens
+
+The **Security Administrator** is able to:
+* Invalidate the token for a user
+* Invalidate all the tokens for a user
+* Invalidate all tokens
+* Evict invalidated tokens and rules which are not relevant anymore
+
+## Generate of the token
+
+The user can create the Personal Access Token by calling the following REST API endpoint via Gateway:
+
+`POST /auth/access-token/generate`
+The full path of the `/auth/access-token/generate` endpoint appear as `https://{gatewayUrl}:{gatewayPort}/gateway/api/v1/auth/access-token/generate`.
+
+The request requires the body in the following format:
+
+```json
+{
+    "validity": 90,
+    "scopes": ["<serviceId_1>,<serviceId_2>"]
+}
+```
+
+The **validity** refers to the expiration time of the tokens. The maximum threshold is 90 days.
+The **scopes** allows to limit the access on a service level, so it will introduce higher level of security in some aspects. Users will be forced to provide a scope and if no service is specified, it will work for none.
+
+When creation is successful, the response to the request is a body containing the PAT. When creation fails, the user receives a 401 status code.
+
+## Validate the token
+
+The user can validate the Personal Access Token by calling the following REST API endpoint via Gateway:
+
+`POST /auth/access-token/validate`
+The full path of the `/auth/access-token/validate` endpoint appear as `https://{gatewayUrl}:{gatewayPort}/gateway/api/v1/auth/access-token/validate`.
+
+The request requires the body in the following format:
+
+```json
+{
+    "tokens": "<token>",
+    "serviceId": "<serviceId>"
+}
+```
+
+When validation is successful, the response to the request is a an empty body. When validation fails, the user receives a 401 status code.
+
+## Invalidate a specific token
+
+The user can invalidate the Personal Access Token by calling the following REST API endpoint via Gateway:
+
+`DELETE /auth/access-token/revoke`
+The full path of the `/auth/access-token/revoke` endpoint appear as `https://{gatewayUrl}:{gatewayPort}/gateway/api/v1/auth/access-token/revoke`.
+
+The request requires the body in the following format:
+
+```json
+{
+  "token": "<token_value>"
+}
+```
+
+When the `/auth/access-token/revoke` endpoint is called, the provided PAT's hash will be stored in the Caching Service will be storing only hashes of already invalidated tokens. Access to these entries is protected by API ML’s client certificate.
+
+When invalidation is successful, the response to the request is an empty body.
```

Comment:
with 200 status code. 

Also I would mention what happens when the response isn't successful. Is there any specific behavior?